src/main/java/uk/co/ramp/covid/simulation/population/Person.java

```diff
@@ -61,6 +65,12 @@ public Person(int age, Sex sex) {
         this.quarantineProb = PopulationParameters.get().personProperties.pQuarantinesIfSymptomatic;
         this.goesToHospitalInPhase2 = CovidParameters.get().hospitalisationParameters.pPhase2GoesToHosptial.sample();
         this.personId = nPeople++;
+        
+        this.covidMortalityAgeAdjustment = Math.pow((double) age / 85.0, 2.0);
+        
+        if(age <= 20) this.covidSusceptibleVal = PopulationParameters.get().personProperties.pSusceptibleChild;
```

Comment:
Should this be <18

Reply:
Same as above, the paper gave parameters for people up to 20